src/app/utils.py

```diff
@@ -203,3 +205,54 @@ def check_license_name(name):
             return [url, exception["licenseExceptionId"]]
 
     return [False]
+
+
+def isConnected():
+    import requests
+    try:
+        response = requests.get("http://www.google.com")
+        return True
+    except requests.ConnectionError:
+        return False
+
+
+def getLicenseList(token):
+    url = "https://api.github.com/"
+    headers = {
+        "Accept":"application/vnd.github.v3.raw+json",
+        "Authorization":"bearer "+token,
+        "Content-Type":"application/json",
+    }
+    license_list_url = url+"repos/spdx/license-list-data/contents/json/licenses.json"
+    response = requests.get(license_list_url, headers=headers)
+    data = json.loads(response.text)
+    return data
+
+
+def licenseInList(namespace, namespaceId, token):
+    license_list = getLicenseList(token)
+    return_dict = {
+    "name": "",
+    "licenseId": "",
+    "referenceNumber": "",
+    "isDeprecatedLicenseId": "",
+    "exists": False
+    }
+    for license in license_list["licenses"]:
+        if namespaceId == license["licenseId"] or namespace == license["name"]:
```

Comment:
You can also compare the license text to see if it matches the listed licenses.  See https://github.com/spdx/spdx-online-tools/blob/ecd4426e8ccdd5ecd79fae10ede31142f900f77a/src/app/views.py#L880 for the code to do this.

Note that  this method is currently rather slow.  I'm planning on making some changes to speed this up over the next month or so.